libcudacxx/include/cuda/std/numbers

```diff
@@ -107,104 +107,103 @@ namespace numbers
 {
 
 template <class _Tp>
-inline constexpr _Tp e_v = __numbers<_Tp>::__e;
+_CCCL_DEVICE_VARIABLE constexpr _Tp e_v = __numbers<_Tp>::__e;
```

Comment:
Why can't we use `_CCCL_GLOBAL_VAR constexpr` here?

Reply:
I thought I had it but I tested the wrong file.

the issue is that we will most likely need to drop MSVC because it cannot have const qualified variable templates 😿

Reply:
Moved to `_CCCL_GLOBAL_CONSTANT` for extended floating point types on windows